.reuse/dep5

```diff
@@ -0,0 +1,30 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: tools-python
+Source: https://github.com/spdx/tools-python
+
+Files:
+  tests/*.json
+  tests/*.xml
+Copyright: 2023 spdx contributors
+License: Apache-2.0
+
+Files: assets/*
```

Comment:
The `assets` folder currently contains a single png image, which is an example output of the graph generation. What license should this have?